css/style.css

```diff
@@ -385,6 +385,12 @@ hr{
     .flex-box{
         flex-flow: column wrap;
     }
+    .welcome .left-panel{
+        order: 1;
+    }
+    .welcome .right-panel{
+        order: 0;
+    }
     h2{
```

Comment:
@ann-qb 
Style for h2 already used( line number 100), so move these additional styles to that.